dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -277,6 +277,22 @@
 			status = "okay";
 		};
 
+		dmic0: dai-dmic@10000 {
+			compatible = "intel,dai-dmic";
+			reg = <0x10000 0x8000>;
+			shim = <0xC000 0x10>;
+			interrupts = <0x08 0 0>;
+			interrupt-parent = <&ace_intc>;
+		};
+
+		dmic1: dai-dmic@10001 {
```

Comment:
yes correct. I just basically want to have 2 entries in dts to be able to instantiate 2 separate dmic dais. In practice it is the 1 and same hw block, but it can operate 2 streams at the same time (we can have for example 48kHz and 16kHz streams from same microphones at the same time).

Historically in sof these have been treated as separate dais, which I still think is the most clear thing to do. People start to get really confused when starting to talk about dmic internal pdm and fifo structures.

If someone has better suggestion, I will change this for sure.

Reply:
@galak @mbolivar @ceolin any suggestions how to do this nicely the "correct way"? see my explanation above.

Reply:
@andyross tried to "clean" the dts by adding separating fifo field into the yaml.... not sure this is better/cleaner.